src/HttpClient.php

```diff
@@ -7,20 +7,30 @@
 
 namespace Garden\Http;
 
-
+/**
+ * Class HttpClient
+ *
+ * @package Garden\Http
```

Comment:
The package comments are not something I like. My rule of thumb is to leave anything that can be reflected out of docs.